tests/conftest.py

```diff
@@ -10,6 +10,8 @@
 os.environ["DVC_TEST"] = "true"
 # Ensure progress output even when not outputting to raw sys.stderr console
 os.environ["DVC_IGNORE_ISATTY"] = "true"
+# Disable system git config
+os.environ["GIT_CONFIG_NOSYSTEM"] = "1"
```

Comment:
dulwich now reads the system-wide default config, which means it will read `core.autocrlf=true` for typical windows Git environments. This should be disabled in CI, as DVC does not read Git configs, and will not do any autocrlf normalization when importing from Git.

Several of our tests git-commit files with CRLF's (usually via Windows `echo` which behaves differently than unix `echo`). If `autocrlf=true`, the original workspace file will have`\r\n` line endings, but the git-committed (and git-index) version will have nomalized `\n` line endings.

If we `dvc import` or `dvc get` that git committed file, DVC will read/download the git-committed version containing `\n` endings (even on Windows). This will break tests on Windows which do a direct comparison between the "original" workspace file and the imported one (since the line endings will not match).